### PR TITLE
Fix metrics map bug

### PR DESCRIPTION
**Pull request by noraarakelyan**

### CODE REVIEW

**Review by a1brt — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cloudchipr/cloudchipr-engine",
-  "version": "0.2.27",
+  "version": "0.2.29",
```
Comment (marashlyansuren) [inline package.json:3]: why not 28 ?

**Review by marashlyansuren — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (noraarakelyan) [reply package.json:3]: 28 is in this pr
https://github.com/cloudchipr/engine/pull/608/files

Comment (marashlyansuren) [reply package.json:3]: Yup, but this gonna be first

Comment (marashlyansuren) [reply package.json:3]: and we cannot just skip

**Pull request merged**